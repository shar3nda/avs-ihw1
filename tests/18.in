11
3 6 -1 8 7 2 3 5 -1 -1 2