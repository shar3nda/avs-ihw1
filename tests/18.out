-3 -7 -6 -2 5 4 -2 8 -8 -3